scripts/ci/exports/utils/module.mjs

```diff
@@ -13,6 +13,12 @@ import { Declaration } from './declaration.mjs';
 import { ExternalModule } from './externalmodule.mjs';
 import { isInternalNode } from './misc.mjs';
 import { createExportResolutionError } from './errorutils.mjs';
+import { globSync } from 'glob';
+import { CKEDITOR5_ROOT_PATH } from '../../../constants.mjs';
+import { PACKAGES_DIRECTORY } from '../../../release/utils/constants.mjs';
+
+// E.g. `packages/ckeditor5-core/` or `packages/ckeditor5/`.
+const PUBLIC_PACKAGES = globSync( `${ PACKAGES_DIRECTORY }/*/`, { cwd: CKEDITOR5_ROOT_PATH, mark: true } );
```

Comment:
We can prepare here an array of public packages in the format we want to use later. Right now `_isFromPublicPackages()` calculates the absolute path for each found file.

```suggestion
const PUBLIC_PACKAGES = globSync( `${ PACKAGES_DIRECTORY }/*/`, {
	cwd: CKEDITOR5_ROOT_PATH,
	mark: true,
	absolute: true
} ).map( upath.normalize );
```